web-frontend/src/project-page/ProjectWidget.svelte

```diff
@@ -134,45 +168,53 @@
     {#if $projectMetadata?.project.startDate}
       <div class=label>Start date</div>
       <div class=data>{$projectMetadata?.project.startDate}</div>
-    {:else if isTestEnvironment}
+    {:else if $isTestEnvironment}
       <div class=label>Start date</div>
       <div class=warning>Start date missing</div>
     {/if}
-    
+
     <!-- End Date -->
     {#if $projectMetadata?.project.endDate}
       <div class=label>End date</div>
       <div class=data>{$projectMetadata?.project.endDate}</div>
     {/if}
 
     <!-- Funders -->
-    {#if $projectMetadata?.project.funders && $projectMetadata?.project.funders.map((o) => {return findObjectByID(o)}).filter(e => e).length>0 }
+    {#if $projectMetadata?.project.funders && $projectMetadata?.project.funders.map((o) => {
+      return findObjectByID(o)
+    }).filter(e => e).length > 0 }
       <div class=label>Funder</div>
-      {#each $projectMetadata?.project.funders.map((o) => {return findObjectByID(o)}) as f}
-          {#if f.__type === "Person"}
-            <!-- TODO: handle funding person - need to find example -->
-            <!-- <div class=data>{findObjectById(f)?.givenName.split(";").join(" ")} {findObjectById(f)?.familyName}</div> -->
-          {:else if f.__type === "Organization"}
-            <div class=data>{f.name}</div>
-          {/if}
+      {#each $projectMetadata?.project.funders.map((o) => {
+        return findObjectByID(o)
+      }) as f}
```

Comment:
Is this done automatically by some kind of formatter? For me this makes entire HTML a bit less readable. I know that some of the lines containing functions are long, but it is either only my feeling, or just entire structure of SVELTE way of HTML code with many statements. My just 2 rappens for later discussion, not something to worry now.

Reply:
I like that maybe you create a followup issue to improve the svelte setup. I would like that.

Reply:
That is good idea. Could you precize what do you like then? Current formatting or previous approach (less line breaks)?